test/Tests/Old.hs

```diff
@@ -99,6 +99,10 @@ tests = [ testGroup "markdown"
           , test "reader" ["-r", "mediawiki", "-w", "native", "-s"]
             "mediawiki-reader.wiki" "mediawiki-reader.native"
           ]
+        , testGroup "vimwiki"
+          [ test "reader" ["-r", "vimwiki", "-w", "html5", "-s"]
```

Comment:
I see above that `test/.vimwiki-reader.wiki.swp` was committed; undoubtedly this is a mistake.